cascade-merge/action.yml

```diff
@@ -0,0 +1,195 @@
+# more info on https://ledgerhq.atlassian.net/l/cp/Ar8F1nmM
+name: 'Auto Cascade Merge'
+description: 'Automatically cascade merge changes from one branch to others.'
```

Comment:
```suggestion
description: 'Automatically cascade merge changes from one branch to others'
```

Reply:
also, I don't think capitalizing all words is standard for `name`, would rather go for `Cascade merge`

Reply:
all applied